Alignment/OfflineValidation/bin/Zmumumerge.cc

```diff
@@ -0,0 +1,310 @@
+#include <iostream>
+#include <TString.h>
+#include <TCanvas.h>
+#include "TBranch.h"
+#include "TTree.h"
+#include "TSystem.h"
+#include "TH1F.h"
+#include "TH1D.h"
+#include "TF1.h"
+#include "TMath.h"
+#include "TLegend.h"
+#include "TLatex.h"
+#include <TH2F.h>
+#include <TCut.h>
+#include <TStyle.h>
+
+#include <TChain.h>
+#include <TFile.h>
+#include <cmath>
+#include <string>
+
+#include "TH1.h"
+#include "TROOT.h"
+#include "TPad.h"
+#include "TRandom.h"
+#include "THStack.h"
+#include "TH2.h"
+#include "TLatex.h"
+#include "TPaveText.h"
+#include "TFrame.h"
+#include "TAxis.h"
+#include <iostream>
+#include <fstream>
+#include <iomanip>
+#include <TCut.h>
+
+#include <RooFit.h>
+#include <RooRealVar.h>
+#include <RooDataHist.h>
+#include <RooDataSet.h>
+#include <RooHistPdf.h>
+#include <RooPlot.h>
+#include <RooAddPdf.h>
+//#include <RooGlobalFunc.h>
+#include <RooGaussian.h>
+#include <RooPolynomial.h>
+#include <RooGenericPdf.h>
+#include <RooFormulaVar.h>
+#include <RooChebychev.h>
+#include <RooExponential.h>
+#include <RooFitResult.h>
+#include <RooWorkspace.h>
+#include <RooMCStudy.h>
+#include "RooHist.h"
+#include "RooConstVar.h"
+#include "RooMsgService.h"
+#include "RooVoigtian.h"
+#include "FitWithRooFit.cc"
+#include "toolbox.h"
+#include "Options.h"
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/optional.hpp>
+
+using namespace RooFit;
+using namespace std;
+using namespace AllInOneConfig;
+namespace pt = boost::property_tree;
+static const int colors_array[] = {4,  8,  2,  3,  4,  7,  30, 6,  9,  46, 36, 8,  10, 11,
+                                   12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24};

```

Comment:
The colours (and the style options in general) should be retrieved from the JSON file. You don't need such an array here.

Reply:
This has been solved, now the color and markerstyle is retrieved from the JSON file.
https://github.com/gu180/TrackderAlignment_DiMuonValidation/tree/all-in-one